### PR TITLE
Adding xy_5 python example

### PR DESCRIPTION
**Pull request by hCraker**
Closes issue #89 
image

### CODE REVIEW

File: Plots/XY/NCL_xy_5.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 4))
+ax = plt.gca()
+
+# Plot reference line
+plt.plot([0, datedim], [0, 0], color='grey', linewidth=0.75)
+
+# Plot data
```
Comment (clairefio) [inline Plots/XY/NCL_xy_5.py:58]: I could be wrong, but it looks like in the original NCL graph, the dsoik line is plotted in addition to the regions being filled. Maybe add this line and then change linewidth to match the thin black borders around the colored regions on the NCL graph:

```suggestion
# Plot data
dsoik.plot(ax=ax, color='black', linewidth=0.75)
```

File: Plots/XY/NCL_xy_5.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# Use geocat.viz.util convenience function to set axes parameters
+gvutil.set_axes_limits_and_ticks(ax, ylim=(-3, 3), 
+                                     yticks=np.linspace(-3, 3, 7),
+                                     yticklabels=np.linspace(-3, 3, 7),
+                                     xlim=(0, datedim),
+                                     xticks=np.arange(0, datedim, 12*20),
+                                     xticklabels=np.arange(1880, 1995, 20))
+
+# Use geocat.viz.util convenience function to set titles and labels
+gvutil.set_titles_and_labels(ax, maintitle="Darwin Southern Oscillation Index", xlabel='', ylabel='')
```
[the remark this answers is not in the document]
Comment (hCraker) [reply Plots/XY/NCL_xy_5.py:76]: If I didn't have that in here, then the x and y labels would automatically be drawn with the names given in the netCDF file. I think there is a more elegant way to do this which I'm trying to find at the moment. This should be cleaner soon.

**Comment by hCraker** [comment]
image

File: Plots/XY/NCL_xy_5.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (clairefio) [reply Plots/XY/NCL_xy_5.py]: Ok, good to know.

File: Plots/XY/NCL_xy_5.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+"""
+NCL_xy_5.py
+===============
+This script illustrates the following concepts:
+   - Adding a separate curve to an XY plot using gsn_polyline
```
Comment (erogluorhan) [inline Plots/XY/NCL_xy_5.py:5]: "gsn_polyline" does not apply to Python code, please revise this concept text.

File: Plots/XY/NCL_xy_5.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+   - Drawing a Y reference line in an XY plot
+   - Filling the areas of an XY curve above and below a reference line
+   - Changing the width and height of a plot
+   - Creating a new date array to use in a plot
+   - Using named colors to indicate a fill color
+   - Changing the title on the Y axis
```
Comment (erogluorhan) [inline Plots/XY/NCL_xy_5.py:11]: Please check if this applies to our Python code

**Pull request merged**